src/ComponentSummary.jsx

```diff
@@ -200,6 +201,34 @@ function normalizeMarkdown(text) {
   return text.trim();
 }
 
+function emitPinnedAppsGatewayEvent(isPinned) {
+  const app = {
+    authorAccountId: accountId,
+    displayName: metadata.name || widgetName,
+    componentName: widgetName,
+    imageUrl: metadata.image?.ipfs_cid
+      ? `https://i.near.social/large/https://ipfs.near.social/ipfs/${metadata.image.ipfs_cid}`
```

Comment:
https://i.near... and https://ipfs... is this the right image url?

Reply:
You're right, this URL does look weird. Let me double check what's going on here.

Reply:
@shelegdmitriy I did some investigating and the URL was working (I had copied it from another place in our codebase). But I went ahead and pushed some changes to use the simpler `https://ipfs.near.social/ipfs/...` version. I can't event remember at this point where `https://i.near.social/large/https://ipfs.near.social...` originally came from.